README.md

```diff
@@ -2,7 +2,7 @@
 The readme-toolkit is a Python repository for generating a readme of a repository.
 
 ## Description
-There is always a tension among the users for writing a flawless readme and it is also very cumbersome to do the documentation in the last moment.
+There is always a tension among the users for writing a flawless readme and it is also very cumbersome to do the documentation in the last moment. This repository has also been made using the this toolkit itself.

```

Comment:
This repository has also been made using ~~the~~ this toolkit itself -> Can you correct the statement